### ISSUE TITLE
Firefox Profiler increases framerate

### ISSUE THREAD

**Issue by tecnd** [renpy/renpyweb#33]
TL;DR: RenPyWeb performs poorly in Firefox, but turning on profiling restores the missing performance, somehow.

So I'm not sure what's happening, or how it makes any sense, or if it's even an issue with RenPyWeb at all, but this seems like the most appropriate place to put it. I'm building for web in RenPy 7.5.2, and it works perfectly in Chrome. Solid 50+ fps basically all the time. However, Firefox can only manage ~20 fps at the best of times. Still playable for a VN, but you can see the text scrolling is pretty choppy. I chalk it up to just browser things and make a note for people to use Chrome. Out of idle curiosity, I open up the Performance tab in the developer tools and start recording to see if I can spot anything strange. The moment I click record, the framerate jumps up to 50. I stop the recording, and it drops back down to 20. I tested this on two different devices on two different VNs, and saw the same behavior. 
Both machines are running Windows 11, Firefox 104.0, one has Intel integrated graphics and the other has a GTX 1080 Ti. RenPy is set to performance mode.
